ods_ci/tests/Resources/Files/pipeline-samples/v2/over_100MB_artifact.py

```diff
@@ -0,0 +1,30 @@
+import os
+from kfp import dsl, compiler
+from kfp.dsl import (component, Output, ClassificationMetrics, Metrics, HTML,
+                     Markdown)
+
+# In tests, we install a KFP package from the PR under test. Users should not
+# normally need to specify `kfp_package_path` in their component definitions.
+_KFP_PACKAGE_PATH = os.getenv('KFP_PACKAGE_PATH')
+
+@component(
+    base_image='registry.redhat.io/ubi8/python-39@sha256:3523b184212e1f2243e76d8094ab52b01ea3015471471290d011625e1763af61',
+    kfp_package_path=_KFP_PACKAGE_PATH,
```

Comment:
Is the value of _KFP_PACKAGE_PATH set?  I did something similar in this [other sample](https://github.com/red-hat-data-services/ods-ci/blob/24083600c088048459d9b208307596d3ac2106ae/ods_ci/tests/Resources/Files/pipeline-samples/v2/pip_index_url/iris_pipeline_pip_index_url.py#L169) and I had to inject the values with kubernetes.use_config_map_as_env